setup.cfg

```diff
@@ -7,8 +7,13 @@ cover-min-percentage=100
 cover-erase=true
 
 [flake8]
+exclude=./docs/conf.py,./test/*,./urllib3/packages/*
 max-line-length=99
 
+[pep8]
```

Comment:
Not needed; I usually do it so people can run pep8 directly and see reasonable results.